source/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(IDE_FOLDER "")
 # Applications
 set(IDE_FOLDER "Applications")
 add_subdirectory(qt-app)
+add_subdirectory(rss-qt-node)
```

Comment:
The naming is a bit confusing. Maybe rename the three to 
- node-qt-rss
- node-lib-qt-rss
- node-lib-cli

Reply:
:+1:

Reply:
fixed in 254e759 and 77d6b2e